src/modules/currency/currencies.ts

```diff
@@ -29,6 +29,17 @@ export function resolveOptions(
   return null;
 }
 
+export function resolveOptions(
+  value: number,
+  config: CurrencyConfig | null,
+  runtimeOptions: Intl.NumberFormatOptions | null
+): Intl.NumberFormatOptions | null {
+  const currencyOptions = resolveCurrencyOptions(value, config);
+  return runtimeOptions
+    ? { ...currencyOptions, ...runtimeOptions }
```

Comment:
Can there be a case where runtime option will override currency option? And can it be bad?

Reply:
That's the point though? :)